types/Post.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from "./User";
+import { Comment } from './Comment';
+
+export interface Post {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+  comments: Comment[] | null;
```

Comment:
If there are no comments it should be an empty array
```suggestion
  comments: Comment[];
```